src/ui/components/TestSuite/views/GroupedTestCases/TestRecordingTreeRow.tsx

```diff
@@ -46,12 +49,12 @@ export default function TestRecordingTreeRow({
       data-test-name="TestRecordingTreeRow"
       onClick={onClick}
     >
-      {showTitle || <Icon className={styles.NestedIcon} type="arrow-nested" />}
+      {showTitleAndError || <Icon className={styles.NestedIcon} type="arrow-nested" />}
       <TestResultIcon result={result} />
       <div className={styles.Column}>
         <div className={styles.HeaderRow}>
           <div className={styles.Title}>
-            {showTitle ? (
+            {showTitleAndError ? (
```

Comment:
I think it's important to retain the attempt information that is now available for playwright. As it stands with this PR, you'll never see the attempt info since the "else" block of the ternary can't be hit for playwright. It probably makes sense to render it a bit differently for playwright anyway since the UX was originally designed to be either the title (for the first attempt) or the attempt number (for subsequent) and that doesn't really make sense for playwright (which only has one attempt in the replay).

I don't think we should block on that change as it's broader than the problem this PR is addressing.